### PR TITLE
system/utils: Update descriptions for shell commands in README.md

### PR DESCRIPTION
**Pull request by jeongchanKim**
Update brief explanations and examples for each commands.

### CODE REVIEW

**Comment by seinfra** [comment]
Code Rule Check OK.

**Comment by seinfra** [comment]
Code Rule Check OK.

**Comment by seinfra** [comment]
Code Rule Check OK.

**Comment by seinfra** [comment]
Code Rule Check OK.

**Comment by seinfra** [comment]
Code Rule Check OK.

**Comment by seinfra** [comment]
Code Rule Check OK.

File: apps/system/utils/README.md [hunk 1]
```diff
@@ -103,15 +113,262 @@ This command supports sleeping of shell. This has an argument which shows sleepi
...
+Feb 27 14:41:06 2018
+```
+Enable *CONFIG_ENABLE_DATE* to use this command on menuconfig as shown below:
+```
+Application Configuration -> System Libraries and Add-Ons -> date
+```
 
 ### dmesg
-### getenv / setenv / unsetenv
+This command shows all debug messages.
```
Comment (sunghan-chang) [inline apps/system/utils/README.md]: This command shows all debug messages which saved in RAM log buffer.

File: apps/system/utils/README.md [hunk 2]
```diff
@@ -103,15 +113,262 @@ This command supports sleeping of shell. This has an argument which shows sleepi
...
+```
+Enable *CONFIG_ENABLE_DMESG* to use this command on menuconfig as shown below:
+```
+Application Configuration -> System Libraries and Add-Ons -> dmesg
+```
+The *CONFIG_SYSLOG* and *CONFIG_RAMLOG_SYSLOG* should be enabled to use this.
+
+### getenv/setenv/unsetenv
+These commands are related with setting or getting the environment variables.  
+getenv prints the environment variable. If there is no environment name, all environments are printed.
```
Comment (sunghan-chang) [inline apps/system/utils/README.md]: The ```**getenv**``` prints all of environment variables. If specific environment name is given, this prints the value of given envionment.

File: apps/system/utils/README.md [hunk 3]
```diff
@@ -103,15 +113,262 @@ This command supports sleeping of shell. This has an argument which shows sleepi
...
+Enable *CONFIG_ENABLE_DMESG* to use this command on menuconfig as shown below:
+```
+Application Configuration -> System Libraries and Add-Ons -> dmesg
+```
+The *CONFIG_SYSLOG* and *CONFIG_RAMLOG_SYSLOG* should be enabled to use this.
+
+### getenv/setenv/unsetenv
+These commands are related with setting or getting the environment variables.  
+getenv prints the environment variable. If there is no environment name, all environments are printed.   
+setenv prints set environment variable.
```
Comment (sunghan-chang) [inline apps/system/utils/README.md]: The ```**setenv**``` save new environment variable with given value.

File: apps/system/utils/README.md [hunk 4]
```diff
@@ -103,15 +113,262 @@ This command supports sleeping of shell. This has an argument which shows sleepi
...
+     4 |      90 |        36 | up_interrupt
+```
+Enable *CONFIG_ENABLE_HEAPINFO* to use this command on menuconfig as shown below:
+```
+Application Configuration -> System Libraries and Add-Ons -> irqinfo
+```
+The *CONFIG_DEBUG* and *CONFIG_DEBUG_IRQ_INFO* should be enabled to use this.
+
+### kill/killall
+These commands sends a signal to specific or all threads.
```
Comment (sunghan-chang) [inline apps/system/utils/README.md]: sends -> send

**Comment by seinfra** [comment]
Code Rule Check OK.

**Comment by seinfra** [comment]
Code Rule Check OK.

File: apps/system/utils/README.md [hunk 5]
```diff
@@ -35,7 +48,273 @@ stkmon           tftpc            ttrace           umount
...
+* Idle Task's stack is not allocated in heap region
+* And another stack size is allocated stack size + alloc node size(task:32/pthread:20)
+```
+### How to Enable
+Enable *CONFIG_ENABLE_HEAPINFO* to use this command on menuconfig as shown below:
+```
+Application Configuration -> System Libraries and Add-Ons -> heapinfo to y
+```
+
+#### Term
```
Comment (sunghan-chang) [inline apps/system/utils/README.md]: This is not a part of ```How to Enable```. Let's move this above ```How to Enable```.

File: apps/system/utils/README.md [hunk 6]
```diff
@@ -35,7 +48,273 @@ stkmon           tftpc            ttrace           umount
...
+    1 |  224 | RR   | KTHREAD |    | WAITSIG  | hpwork
+    3 |  125 | RR   | TASK    |    | RUNNING  | tash
+```
+### How to Enable
+Enable *CONFIG_ENABLE_PS* to use this command on menuconfig as shown below:
+```
+Application Configuration -> System Libraries and Add-Ons -> ps to y
+```
+
+#### Term
```
Comment (sunghan-chang) [inline apps/system/utils/README.md]: same as previous

**Comment by seinfra** [comment]
Code Rule Check OK.

**Pull request merged**